src/components/LineChart/LineChart.jsx

```diff
@@ -371,12 +380,14 @@ const LineChart = createClass({
 			yAxisTitleColor,
 			yAxisMin,
 			yAxisTooltipFormatter,
+			yAxisTooltipDataFormatter,
```

Comment:
And it doesn't seem like you can set `yAxisTooltipDataFormatter: yAxisTooltipFormatter,`